### ISSUE TITLE
Tomcat error banner

### ISSUE THREAD

**Issue by drwetter** [sullo/nikto#500]
It doesn't show up in the normal output (Display: 4) -- but it does e.g. during Debug (D) output.

As the error contains valuable version info as opposed to " Apache-Coyote/1.1" in the HTTP header it would be great to see this in normal output, too.

Version used: master

**Comment by sullo** [comment]
Do you have a sample of it?

**Comment by drwetter** [comment]
Sure, here you go

```
<!DOCTYPE html><html><head><title>Apache Tomcat/8.0.20 - Error report</title><style type="text/css">H1 {font-family:Tahoma,Arial,sans-serif;color:
white;background-color:#525D76;font-size:22px;} H2 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:16px;} H3 {font-family:Tahoma,Arial,sa
ns-serif;color:white;background-color:#525D76;font-size:14px;} BODY {font-family:Tahoma,Arial,sans-serif;color:black;background-color:white;} B {font-family:Tahoma,Arial,san
s-serif;color:white;background-color:#525D76;} P {font-family:Tahoma,Arial,sans-serif;background:white;color:black;font-size:12px;}A {color : black;}A.name {color : black;}.
line {height: 1px; background-color: #525D76; border: none;}</style> </head><body><h1>HTTP Status 404 - /8SHUhZVY.jsp</h1><div class="line"></div><p><b>type</b> Status repor
t</p><p><b>message</b> <u>/8SHUhZVY.jsp</u></p><p><b>description</b> <u>The requested resource is not available.</u></p><hr class="line"><h3>Apache Tomcat/8.0.20</h3></body>
</html>
```

This is captured from nikto's debug output. (see resource requested). I have a bourne shell script which searches for `` Tomcat.[0-9]\.[0-9]\.[0-9][0-9]``.

**Comment by drwetter** [comment]
For Tomcat 7.x it looks like

```
doctype html><html lang="en"><head><title>HTTP Status 404 – Not Found</title><style type="text/css">H1 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:22px;} H2 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:16px;} H3 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:14px;} BODY {font-family:Tahoma,Arial,sans-serif;color:black;background-color:white;} B {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;} P {font-family:Tahoma,Arial,sans-serif;background:white;color:black;font-size:12px;}A {color : black;}A.name {color : black;}HR {color : #525D76;}</style></head><body><h1>HTTP Status 404 – Not Found</h1><hr class="line" /><p><b>Type</b> Status Report</p><p><b>Description</b> The origin server did not find a current representation for the target resource or is not willing to disclose that one exists.</p><hr class="line" /><h3>Apache Tomcat/7.0.92</h3></body></html>
```

I would additionally match on ``Apache .*.[0-9]\.[0-9]\.[0-9][0-9].`` (probably no perl regex too)

**Comment by sullo** [comment]
Resolved in 805bcebd03a5c3b17c6ea5ec6ad39a7d41740e01 by adding content check.
